lrauv_ignition_plugins/test/test_spawn.cc

```diff
@@ -138,20 +147,20 @@ TEST(SpawnTest, Spawn)
   // Check vehicle positions
   EXPECT_NEAR(0.0, poses1.back().Pos().X(), 1e-3);
   EXPECT_NEAR(0.0, poses1.back().Pos().Y(), 1e-3);
-  EXPECT_NEAR(0.0, poses1.back().Pos().Z(), 1e-3);
+  EXPECT_NEAR(0.0, poses1.back().Pos().Z(), 1e-2);
   EXPECT_NEAR(0.0, poses1.back().Rot().Roll(), 1e-3);
-  EXPECT_NEAR(0.0, poses1.back().Rot().Pitch(), 1e-3);
+  EXPECT_NEAR(0.0, poses1.back().Rot().Pitch(), 1e-2);
   EXPECT_NEAR(0.0, poses1.back().Rot().Yaw(), 1e-3);
 
   EXPECT_NEAR(20.0, latLon1.back().X(), 1e-3);
   EXPECT_NEAR(20.0, latLon1.back().Y(), 1e-3);
-  EXPECT_NEAR(0.0, latLon1.back().Z(), 1e-3);
+  EXPECT_NEAR(0.0, latLon1.back().Z(), 1e-2);
 
   EXPECT_LT(100.0, poses2.back().Pos().X());
   EXPECT_LT(100.0, poses2.back().Pos().Y());
   EXPECT_GT(0.0, poses2.back().Pos().Z());
   EXPECT_NEAR(0.0, poses2.back().Rot().Roll(), 1e-3);
-  EXPECT_NEAR(0.0, poses2.back().Rot().Pitch(), 1e-3);
+  EXPECT_NEAR(0.0, poses2.back().Rot().Pitch(), 1e-2);
```

Comment:
Are these specific axes different in terms of the noise. I would suggest that we just have a single tolerance for individual semantically meaningful values like linear and angular errors.

Reply:
I agree that it's useful to have semantically meaningful tolerances. I split them differently from your suggestion on #64 though, see the added comment and let me know what you think: 067cfd9